src/codegen/llvm.zig

```diff
@@ -1791,6 +1811,27 @@ pub const Object = struct {
             "", // unique id
         );
     }
+
+    fn getStackTraceType(o: *Object) Type {
```

Comment:
This looks a little fishy. I think it's OK for now, but before this function gets copy+pasted into the other backends, let's look into moving this logic into Sema. Perhaps for a Module in which error return tracing is enabled, it can create a root reference to this type and store it directly in Module.